packages/docs-page/props.js

```diff
@@ -69,20 +69,73 @@ module.exports = {
-            '}\n' +
+            `function MDXContent(_ref) {
+              var components = _ref.components,
+                props = _objectWithoutProperties(_ref, ["components"]);
+              return mdx(
+                MDXLayout,
+                _extends({}, layoutProps, props, {
+                  components: components,
+                  mdxType: "MDXLayout",
+                }),
+                mdx("h1", { className: "g-type-display-2" }, "Example Page"),
+                mdx("p", null, "This is a cool docs page!"),
+                mdx(
+                  "h2",
+                  { className: "g-type-display-3" },
+                  mdx(
+                    "a",
+                    { className: "__permalink-h", href: "#second-heading" },
+                    "»"
+                  ),
+                  mdx(
+                    "a",
+                    { className: "__target-h", id: "second-heading", "aria-hidden": true },
+                    ""
+                  ),
+                  "Second heading"
+                ),
+                mdx("p", null, "This is a cool docs page!"),
+                mdx(
+                  "h2",
+                  { className: "g-type-display-3" },
+                  mdx(
+                    "a",
+                    { className: "__permalink-h", href: "#third-heading" },
+                    "»"
+                  ),
+                  mdx(
+                    "a",
+                    { className: "__target-h", id: "third-heading", "aria-hidden": true },
+                    ""
+                  ),
+                  "Third heading"
+                ),
+                mdx("p", null, "This is a cool docs page!"),
+                mdx(
+                  "h2",
+                  { className: "g-type-display-3" },
+                  mdx(
+                    "a",
+                    { className: "__permalink-h", href: "#fourth-heading" },
+                    "»"
+                  ),
+                  mdx(
+                    "a",
+                    { className: "__target-h", id: "fourth-heading", "aria-hidden": true },
+                    ""
+                  ),
+                  "Fourth heading"
+                ),
+                mdx("p", null, "This is a cool docs page!")
+              );
+            }
+            \n` +
             '\n' +
             ';\n' +
             'MDXContent.isMDXComponent = true;',
           renderedOutput:
-            '<h1 className="g-type-display-2">Example Page</h1><p>This is a cool docs page!</p>',
+            '<h1 className="g-type-display-2">Example Page</h1><p>This is a cool docs page!</p><h2 className="g-type-display-3">Second Heading</h2><p>This is a cool docs page!</p><h2 className="g-type-display-3">Third Heading</h2><p>This is a cool docs page!</p><h2 className="g-type-display-3">Fourth Heading</h2><p>This is a cool docs page!</p>',
```

Comment:
Updated to reflect the updated fixture. Related Asana task to make this type of task easier: https://app.asana.com/0/1100423001970639/1200356179054754/f .